capstone/src/main/java/com/google/sps/servlets/announcements/AnnouncementsServlet.java

```diff
@@ -0,0 +1,87 @@
+package com.google.sps.servlets;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Streams;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example club content */
+@WebServlet("/announcements")
+public class AnnouncementsServlet extends HttpServlet {
+
+  private static final String ANNOUNCEMENTS_CONTENT_PROP = "content";
+  private static final String ANNOUNCEMENT_PROP = "Announcement";
+  private static final String AUTHOR_PROP = "author";
+  private static final String TIME_PROP = "time";
+  private static final String CONTENT_PROP = "content";
+  private static final String CLUB_PROP = "club";
```

Comment:
I may have missed if this was alreday brought up, but if this is used in multiple places maybe move it to a constants file